CesiumGltf/include/CesiumGltf/MetadataConversions.h

```diff
@@ -441,20 +560,13 @@ template <> struct MetadataConversions<double, std::string_view> {
       // misleading.
       return std::nullopt;
     }
+
     // Amazingly, C++ has no* string parsing functions that work with strings
     // that might not be null-terminated. So we have to copy to a std::string
     // (which _is_ guaranteed to be null terminated) before parsing.
     // * except std::from_chars, but compiler/library support for the
     //   floating-point version of that method is spotty at best.
-    std::string temp(from);
-
-    char* pLastUsed;
-    double parsedValue = std::strtod(temp.c_str(), &pLastUsed);
-    if (pLastUsed == temp.c_str() + temp.size() && !std::isinf(parsedValue)) {
-      // Successfully parsed the entire string as a double.
-      return parsedValue;
-    }
-    return std::nullopt;
+    return MetadataConversions<double, std::string>::convert(std::string(from));
```

Comment:
The comment block preceding this line needs to be moved to line 535.

Reply:
This comment block initially annotated the conversion from `std::string_view` -> `double`. The point was that, due to a strange bug, you could not convert from the `std::string_view` directly and had to copy the data to a `std::string`. This is why I kept it in the `std::string_view` function, because the `std::string_view` must still be converted to a `std::string` before being converted to a float.